Protocols/IEC61850/SWDevice-010805/StartSelfTest.md

```diff
@@ -0,0 +1,148 @@
+## StartSelfTest messages
+
+### Description
+
+Request which commands the device to switch all light relays on and then queries the device for the status of the relays.
+Part of this request is also to retrieve the status of the relays after switching on.
```

Comment:
More or less duplicates "and then queries the device for the status of the relays", which is on the previous line.